doc/nrf/glossary.rst

```diff
@@ -461,6 +468,9 @@ Glossary
       A feature introduced in 3GPP Release 12 to improve the battery life of IoT (Internet of Things) devices by minimizing energy consumption.
       The device stays dormant during the PSM window.
 
+   Preconnect Provisioning
+      The process of securely generating and storing credentials in a device, then uploading a device id and device certificate to a cloud account so that the device is ready to connect to the cloud.
```

Comment:
```suggestion
   Preconnect provisioning
      The process of securely generating and storing credentials in a device, then uploading a device ID and device certificate to a cloud account so that the device is ready to connect to the cloud.
```